### PR TITLE
Feat: Lt 980 edit company name for organisation

### PR DESCRIPTION
**Pull request by stephanieboyd**
- Add edit organisation

### CODE REVIEW

File: organisations/views.py [hunk 1]
```diff
@@ -125,3 +138,27 @@ def post(self, request, **kwargs):
 
         messages.success(request, strings.HMRC_ORGANISATION_CREATION_SUCCESS)
         return redirect("organisations:hmrc")
+
+
+class EditBusiness(MultiFormView):
+    forms = None
+
+    def __init__(self, **kwargs):
```
Comment (Mark-Spiers-kainos) [inline organisations/views.py]: I don't like having two inits, especially when it can all be moved into one.

File: organisations/forms.py [hunk 2]
```diff
@@ -119,3 +120,76 @@ def register_hmrc_organisation_forms(name=""):
...
+                            Option(key="commercial", value="Commercial"),
+                            Option(key="individual", value="Individual"),
+                        ],
+                    )
+                ],
+                default_button_name=strings.CONTINUE,
+            ),
+            conditional(
+                individual,
+                Form(
```
Comment (Mark-Spiers-kainos) [inline organisations/forms.py]: perhaps move the forms inside the conditional to separate functions, to be a bit easier to read.

File: organisations/forms.py [hunk 3]
```diff
@@ -119,3 +120,76 @@ def register_hmrc_organisation_forms(name=""):
...
+                    title=strings.RegisterBusiness.EDIT_COMMERCIAL_TITLE,
+                    questions=[
+                        conditional(
+                            permission_to_edit_org_name,
+                            TextInput(title=strings.REGISTER_BUSINESS_FIRST_AND_LAST_NAME, name="name"),
+                        ),
+                        TextInput(title=strings.RegisterBusiness.EORI_NUMBER, name="eori_number"),
+                        TextInput(
+                            title=strings.RegisterBusiness.SIC_NUMBER,
+                            description="Classifies industries by a four-digit code.",
```
Comment (andytaylorkainos) [inline organisations/forms.py]: What is dis string

Comment (stephanieboyd) [reply organisations/forms.py]: Fixed. It was actually duplicated, so I added the correct string

**Pull request merged**